src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Administration/GeneralType.php

```diff
@@ -43,19 +45,52 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('check_modules_update', SwitchType::class, [
-                'required' => true,
+                'label' => $this->trans('Automatically check for module updates', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('New modules and updates are displayed on the modules page.', 'Admin.Advparameters.Help'),
             ])
             ->add('check_ip_address', SwitchType::class, [
-                'required' => true,
+                'label' => $this->trans('Check the cookie\'s IP address', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('Check the IP address of the cookie in order to prevent your cookie from being stolen.', 'Admin.Advparameters.Help'),
             ])
             ->add('front_cookie_lifetime', TextType::class, [
-                'required' => true,
+                'label' => $this->trans('Lifetime of front office cookies', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('Set the amount of hours during which the front office cookies are valid. After that amount of time, the customer will have to log in again.', 'Admin.Advparameters.Help'),
+                'constraints' => [
+                    new Type(
+                        [
+                            'value' => 'numeric',
+                            'message' => $this->trans('The field is invalid', 'Admin.Notifications.Error'),
```

Comment:
There is a string that already exists "The %s field is invalid." in Admin.Notifications.Error Couldn't we use this one?

Otherwise, we also have "This field is invalid." in Admin.Notifications.Error.